src/codeu/chat/server/Server.java

```diff
@@ -221,20 +260,15 @@ public void run() {
     });
   }
 
-  public User newUser(String id, String name, String time){
-	  
-	  
-	  Uuid userid = new Uuid(Integer.parseInt(id));
+  //adds new user at the start
+  public void addNewUser(String id, String name, String time){
 	  
+	  //converts strings to necessary objects
+	  Uuid userid = new Uuid(Integer.parseInt(id));	  
 	  Time usercreation = Time.fromMs(Long.parseLong(time));
 	  
+	  this.controller.newUser(userid, name, usercreation);
 	  

```

Comment:
I just tried this out and the code works!! The only difference was that the Uuid goes from 1.333->.333 (essentially the one before the decimal gets taken out, but I'm not entirely sure why?) :)